catkin_virtualenv/src/dh_virtualenv/deployment.py

```diff
@@ -98,12 +102,21 @@ def __init__(self,
         self.pip_args.extend([
             '--extra-index-url={0}'.format(url) for url in extra_urls
         ])
-        self.pip_args.append('--log={0}'.format(os.path.abspath(self.log_file.name)))
+
+        # (pbovbel): make logging optional, since --log overrides -q flag for pip
+        if log_file:
+            self.log_file = log_file
```

Comment:
I would assign `self.log_file=None` when `log_file` is false-y, to avoid having the exposed attributes change based on its `__init__` arguments.

Reply:
Done